### PR TITLE
DEV: updated search content

### PR DESCRIPTION
**Pull request by dwdougherty**
Ticket: [DOC-3764](https://redislabs.atlassian.net/browse/DOC-3764)

@adrianoamaral This is the lion's share of the search content for RediSearch v.2.10/Redis 7.6-rc1. The only item missing is updates for the INFO command. I'll add that once I have everything I need.

[DOC-3764]: https://redislabs.atlassian.net/browse/DOC-3764?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by dwdougherty** [comment]
Opening up this PR for peer review.

File: content/develop/interact/search-and-query/basic-constructs/configuration-parameters.md [hunk 1]
```diff
@@ -212,6 +213,22 @@ $ redis-server --loadmodule ./redisearch.so MINPREFIX 3
 
 ---
 
+### MINSTEMLEN
+
+The minimum word length to stem. The default value is `4`. Setting it lower than `4` can hurt performance.
```
Comment (mich-elle-luna) [inline content/develop/interact/search-and-query/basic-constructs/configuration-parameters.md:218]:
```suggestion
The minimum word length to stem. The default value is `4`. Setting it lower than `4` can reduce performance.
```

**Review by mich-elle-luna — APPROVED** [review]
Just one small change for clarity.

File: content/develop/interact/search-and-query/basic-constructs/configuration-parameters.md (hunk 1 above)
Comment (dwdougherty) [reply content/develop/interact/search-and-query/basic-constructs/configuration-parameters.md:218]: Thank you, @mich-elle-luna ! I'll push the change from my system.

**Pull request merged**